share-ami.sh

```diff
@@ -50,7 +51,7 @@ share_ami() {
     done
 
     # Update the image tags
-    local tags="$(jq -n --arg build_date "$build_date" --arg version "$TAUPAGE_VERSION" --arg source_ami "$imageid" --arg commit_id "$commit_id" '[{Key: "BuildDate", Value: $build_date}, {Key: "SourceAMI", Value: $source_ami}, {Key: "CommitID", Value: $commit_id}, {Key: "Version", Value: $version}]')"
+    local tags="$(jq -n --arg build_date "$build_date" --arg version "$TAUPAGE_VERSION" --arg source_ami "$imageid" --arg commit_id "$commit_id" --arg ami_expiration "$ami_expiration" '[{Key: "BuildDate", Value: $build_date}, {Key: "SourceAMI", Value: $source_ami}, {Key: "CommitID", Value: $commit_id}, {Key: "Version", Value: $version}, {Key: "ExpireDate", Value: $ami_expiration}]')"
```

Comment:
The tag is called `ExpirationTime`: https://github.bus.zalan.do/torch/fullstop/blob/f1f101f684f5a28628ed7cc794cfffd06e413df4/fullstop-jobs/src/main/java/org/zalando/stups/fullstop/jobs/common/impl/TaupageExpirationTimeProviderImpl.java#L21

Reply:
Thanks!